src/config.hpp

```diff
@@ -474,6 +474,12 @@ class config
 	 */
 	const attribute_value *get(config_key_type key) const;
 
+    /**
+     * Chooses a value. If the value specified by @a key is
+     * blank, then @a default_key is chosen instead.
+     */
+    const attribute_value &get_or(const config_key_type key, const config_key_type default_key)
```

Comment:
Can probably be a `const` method, and is missing the semicolon.

Reply:
`const attribute_value& get_or`